### PR TITLE
webrtc: add test assertions for simulcast-~ spec change

### PR DESCRIPTION
**Pull request by fippo**
see https://github.com/w3c/webrtc-pc/issues/2735

### CODE REVIEW

**Comment by fippo** [comment]
spec change in https://github.com/w3c/webrtc-pc/pull/2754
@henbos i'll deal with the chrome failures after import

File: webrtc/protocol/simulcast-answer.html [hunk 1]
```diff
@@ -57,6 +56,42 @@
...
+  // Try to disable the `bar` encoding by prefixing it with the `~` character.
+  await pc.setRemoteDescription({type: 'offer', sdp: offer_sdp.replace('bar', '~bar')});
+  transceiver.direction = 'sendonly';
+  await pc.setLocalDescription();
+
+  const parameters = pc.getSenders()[0].getParameters();
+  const barEncoding = parameters.encodings.find(encoding => encoding.rid === 'bar');
+  assert_not_equals(barEncoding, undefined);
+  assert_not_equals(barEncoding.active, false);
+}, 'Using the ~rid SDP syntax in a remote description does not control the local encodings active flag');
```
Comment (jan-ivar) [inline webrtc/protocol/simulcast-answer.html:76]: Do we need a similar test for answer?
```suggestion
}, 'Using the ~rid SDP syntax in a remote offer does not control the local encodings active flag');
```

File: webrtc/protocol/simulcast-answer.html [hunk 2]
```diff
@@ -57,6 +56,42 @@
...
+  // Disabling the encoding should not change the rid to ~rid.
+  const parameters = pc.getSenders()[0].getParameters();
+  parameters.encodings.forEach(e => e.active = false);
+  await pc.getSenders()[0].setParameters(parameters);
+  const offer = await pc.createOffer();
+
+  let result = answer_lines.find(
+      line => line.startsWith(`a=simulcast:send ${expected_rids.join(';')}`));
+  assert_not_equals(result, undefined, 'Could not find simulcast attribute.');
+}, 'Using the ~rid SDP syntax in a remote description does not control the local encodings active flag');
```
Comment (jan-ivar) [inline webrtc/protocol/simulcast-answer.html:96]: update description to match test

**Review by jan-ivar — APPROVED** [review]
Looks good, but should some of these perhaps be in `simulcast-offer.html` instead?

I ask because I'd like to reference them in https://github.com/w3c/webrtc-pc/pull/2755.

**Comment by fippo** [comment]
the -offer path is a bit more complicated sadly :-(

**Comment by youennf** [comment]
Test failure in STP happens with and without the changes so good to go.

**Pull request merged**